corehq/apps/app_manager/xpath.py

```diff
@@ -45,6 +46,9 @@ def interpolate_xpath(string, case_xpath=None):
         '#user': UserCaseXPath().case(),
         '#session/': session_var('', path=''),
     }
+    if module:
```

Comment:
If that check should happen then it can happen outside the function and the function param can change to `fixture_xpath` (similar to `case_xpath`)